scripts/automation/Radius/Functions/Private/settings/Set-JCRSettingsFile.ps1

```diff
@@ -0,0 +1,100 @@
+function Set-JCRSettingsFile {
+    [CmdletBinding()]
+    param (
```

Comment:
Is param() needed here if it is not used?

Reply:
This one is kind of strange but you do need a null param set if even if your function only has dynamic parameters so in this instance, we do need this :)